test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -171,10 +211,17 @@ private static string EnsureTestData(Options opts, string workingFolder)
                 : opts.Branch;
             if (!Directory.Exists(workingFolder))
             {
-                Directory.CreateDirectory(workingFolder);
-                Exec("git", $"init", cwd: workingFolder);
-                Exec("git", $"remote add origin {TestDataRepositoryUrl}", cwd: workingFolder);
-                Exec("git", $"{s_gitCmdAuth} fetch origin --progress template", cwd: workingFolder, secrets: s_gitCmdAuth);
+                try
+                {
+                    Directory.CreateDirectory(workingFolder);
+                    Exec("git", $"init", cwd: workingFolder);
+                    Exec("git", $"remote add origin {TestDataRepositoryUrl}", cwd: workingFolder);
+                    Exec("git", $"{s_gitCmdAuth} fetch origin --progress template", cwd: workingFolder, secrets: s_gitCmdAuth);
+                }
+                catch
```

Comment:
When will this throw? This could easily leads to infinite loop.

Reply:
Sometimes the url `https://dev.azure.com/ceapex/Engineering/_git/docfx.TestData` return a 503 code.

Reply:
Consider catch and retry the `fetch` command for a finite time (like 3), delay a second or so between each retry. The current approach may leads to infinite loop for non-retriable exception.

Reply:
Fixed.